_data/advisories.yml

```diff
@@ -1,3 +1,7 @@
+- advisory: 50587
+  summary: TRUNCATE prevents table renaming
+  versions: 2.1.0-2.1.9, 19.1.0-19.1.10, 19.2.0 - 19.2.8
```

Comment:
I don't think we care about 2.1.x anymore, I'd safely remove, especially that a backport to 2.1 is not happening.